### PR TITLE
Receive/fallback functions in UI

### PR DESCRIPTION
**Pull request by LianaHus**

### CODE REVIEW

File: src/app/ui/multiParamManager.js [hunk 1]
```diff
@@ -204,12 +204,15 @@ class MultiParamManager {
...
       contractProperty.classList.add(css.hasArgs)
-    } else if (this.funABI.type === 'fallback') {
-      contractProperty.classList.add(css.hasArgs)
-      this.contractActionsContainerSingle.querySelector('i').style.visibility = 'hidden'
     } else {
       this.contractActionsContainerSingle.querySelector('i').style.visibility = 'hidden'
-      this.basicInputField.style.visibility = 'hidden'
+      if (this.funABI.type === 'fallback' || this.funABI.type === 'receive') {
+        contractProperty.classList.add(css.hasArgs)
+        this.basicInputField.setAttribute('title', `'(${this.funABI.type}')`) // probably should pass name instead
```
Comment (LianaHus) [inline src/app/ui/multiParamManager.js]: this seems wrong I'll fix

File: src/app/ui/universal-dapp-ui.js [hunk 2]
```diff
@@ -131,172 +115,160 @@ UniversalDAppUI.prototype.renderInstanceFromABI = function (contractABI, address
...
+    <label id="deployAndRunLLTxError" class="text-danger"></label>
+  `
+  // constract LLInteractions elements
+  const lowLevelInteracions = yo`
+    <div class="d-flex flex-column">
+      <div class="d-flex flex-row justify-content-between mt-2">
+        <label class="pt-2 border-top d-flex justify-content-start flex-grow-1">
+          Low level interactions with contract
+        </label>
+        <i aria-hidden="true"
```
Comment (yann300) [inline src/app/ui/universal-dapp-ui.js]: can we have a link to doc instead of this info?

File: src/app/ui/universal-dapp-ui.js [hunk 3]
```diff
@@ -131,172 +115,160 @@ UniversalDAppUI.prototype.renderInstanceFromABI = function (contractABI, address
...
+      if (calldata.length > 3 && calldata.substr(0, 2) === '0x') {
+        if (!helper.isHexadecimal(calldata.substr(2, calldata.length))) {
+          setLLIError('the calldata should be a valid hexadecimal value.')
+        }
+      }
+      if (!fallback) {
+        setLLIError("'fallback' function is not defined")
+      }
+    }
+    if ((calldata || amount !== '0') && !error) self.runTransaction(false, args, null, calldata, null)
```
Comment (yann300) [inline src/app/ui/universal-dapp-ui.js]: why `amount !== '0'`

can't you just do `return setLLIError(...)`  when check fails and `self.runTransaction(false, args, null, calldata, null)` at the end?

File: src/app/ui/universal-dapp-ui.js (hunk 2 above)
Comment (LianaHus) [reply src/app/ui/universal-dapp-ui.js]: ah, sorry about this. totally forgot.

File: src/app/ui/universal-dapp-ui.js (hunk 3 above)
Comment (LianaHus) [reply src/app/ui/universal-dapp-ui.js]: and send an 'empty' transaction?

Comment (yann300) [reply src/app/ui/universal-dapp-ui.js]: what do you mean empty?
even if there is no data and no value, that's still a valid tx. @LianaHus

Comment (LianaHus) [reply src/app/ui/universal-dapp-ui.js]: lets do this in the second pr

Comment (yann300) [reply src/app/ui/universal-dapp-ui.js]: please include this now (that's more readable)

**Pull request merged**